packages/react/src/components/FilterTags/FilterTags.jsx

```diff
@@ -78,6 +90,11 @@ const FilterTags = ({ children, hasOverflow, id, tagContainer, onChange, i18n, t
     /* eslint-enable react-hooks/exhaustive-deps */
   );
 
+  const filterTagsOverflowMenuTextText = filterTagsOverflowMenuText.replace(
```

Comment:
<img width="1275" alt="image" src="https://user-images.githubusercontent.com/18572477/217228607-4d81cbb7-f846-4cf5-bf20-dcfd068d8140.png">
better to use it as a callback function. so if they need any format they can use.

Reply:
@herleraja Thanks for the tip! I replaced string with the callback function.